solver/src/solver/baseline_solver.rs

```diff
@@ -59,42 +59,76 @@ enum AmmOrder {
     WeightedProduct(WeightedProductOrder),
 }
 
+impl BaselineSolvable for ConstantProductOrder {
```

Comment:
This was done to be able to call `BaselineSolvable` methods directly on `ConstantProductOrder`s.